manifests/init.pp

```diff
@@ -213,4 +309,14 @@
       accept  => '127.0.0.1:873'
     }
   }
+
+  ### MOVE TO PUPMOD?
```

Comment:
We need to see where this is used. I *think* that it's used in multiple locations and should stay here.